### PR TITLE
Add override

### LINKED ISSUE #454: Highlight current folder on aggregate pages (left hand menu)
For routes like `/flow` and `/nodes`. The left menu should highlight the current folder that the page being shown is (which should be the first one in the menu).

### PR DESCRIPTION
**Pull request by aureentuluvacal**
Closes #454 

This seems a little weird to me, but I can't come up with a more extensible way of doing this without a complete overhaul of the sidebar component. I don't think that would be worth it.

image

image

### CODE REVIEW

**Review by 10thfloor — APPROVED** [review]

**Comment by joekrill** [comment]
Yeah I think we should rethink what we're doing here in general. It might be best to simply redirect `/node` to `/nodes/node-operation` and redirect `/flow `/flow/dapp-development` (and TBH I thought that's what we were planning on doing here). Otherwise we will have 2 pages rendering the same exact content (but with the sidebar being different). This is going to be confusing for users, and I suspect will also be problematic for search indexing (because you'll see two results with the same content/title). 

Either that or we should have some distinct content for `/flow` and `/nodes` that acts like more of an overview page.

@10thfloor any thoughts on this? Perhaps Alex has some input (not sure of his github name, if he has one, to @-mention him).

**Comment by 10thfloor** [comment]
I think we're working on landing pages for these so we should redirect for now.

**Review by 10thfloor — APPROVED** [review]

File: app/ui/design-system/src/lib/Components/InternalSidebar/index.tsx [hunk 1]
```diff
@@ -60,10 +60,14 @@ export const TEMP_SIDEBAR_CONFIG: InternalSidebarConfig = {
 }
 
 export function InternalSidebar({ config, menu }: InternalSidebarProps) {
+  const location = useLocation()
+  const itemOverride =
+    location.pathname === "/flow/" || location.pathname === "/nodes/"
```
Comment (10thfloor) [inline app/ui/design-system/src/lib/Components/InternalSidebar/index.tsx:65]: Hate to see this, but seems unavoidable with the current setup. Going tohold of on merging this until @joekrill finishes up with the routing changes.

**Comment by joekrill** [comment]
I don't _think_ this will be necessary anymore - but i'm not 100% sure. In the refactor changes each collection can specify a `redirects` section which is just a mapping of paths to redirect (i.e. 016c3486eefa795cb9ba51025b90f486997c870e4ca7e2106377e11dc070R100). I didn't add these particular redirects for some reason, but maybe we should handle that there instead?

**Comment by joekrill** [comment]
Closing this because I believe this has been implemented in other ways since the routing refactor.

**Pull request closed without merging**